packages/tools/src/enums/index.js

```diff
@@ -13,4 +15,5 @@ export {
   Events,
   SegmentationRepresentations,
   Swipe,
+  CalibrationTypes,
```

Comment:
I don't understand why we are re import/exporting this here, since we already have it from core